tasks/swig.rake

```diff
@@ -13,22 +15,23 @@ def run_swig(mod)
   swig = `which swig2.0`.chomp if swig.empty?
 
   # Standard search location for headers
-  include_args = %w[-I/usr/local/include -I/usr/include]
+  include_args = "-I#{Taglib.install_dir}/include"
 
   if ENV.key?('TAGLIB_DIR')
```

Comment:
I wonder if we should just ignore TAGLIB_DIR in `rake swig`?

Reply:
Hm I think it might still be useful to be able to use a different version temporarily (maybe to check for differences between different versions?)

Reply:
What bothers me is that CI now overrides the taglib version used by `rake swig`, because CI sets TAGLIB_DIR.

So the real problem is that CI uses a completely different environment from local development.

The reason to have TAGLIB_DIR is for our users because we can't know where their TagLib is installed.

Reply:
>maybe to check for differences between different versions?

If you want to check for `rake swig` differences you might as well edit what is now `tasks/build.rb` and run `git diff`.

Reply:
I'm starting to think that if we get to the point where CI fails if `*_wrap.cxx` changes it's probably all fine. Let's leave this alone for now.